src/test/java/seedu/address/model/student/AddressTest.java

```diff
@@ -19,6 +20,12 @@ public void constructor_invalidAddress_throwsIllegalArgumentException() {
         assertThrows(IllegalArgumentException.class, () -> new Address(invalidAddress));
     }
 
+    @Test
+    public void hashcode_equals() {
+        String address = "Blk 123 Outer space road";
+        assertEquals(new Address(address).hashCode(), new Address(address).hashCode());
```

Comment:
does this and the remaining hashcode test cases somehow increase coverage? hahaha